mobile/lib/components/ramowka/fetch_ramowka.dart

```diff
@@ -1,21 +1,26 @@
 import 'dart:async';
 
+import 'package:radioaktywne/extensions/extensions.dart';
 import 'package:radioaktywne/models/ramowka_info.dart';
 import 'package:radioaktywne/resources/fetch_data.dart';
-
-final Uri url = Uri.parse(
-  'https://radioaktywne.pl/wp-json/wp/v2/event?_embed=true&page=1&per_page=100',
-);
-const headers = {'Content-Type': 'application/json'};
+import 'package:radioaktywne/resources/ra_links.dart';
 
 /// A conviniance wrapper around [fetchData] for fetching
```

Comment:
If we're already at it we can correct this too 😅
```suggestion
/// A convenient wrapper around [fetchData] for fetching
```